lib/pbench/test/unit/server/test_api_key.py

```diff
@@ -125,28 +140,59 @@ def test_successful_api_key_get(
         pbench_drb_api_key,
         pbench_drb_secondary_api_key,
     ):
-
         response = query_get_as(pbench_drb_token, HTTPStatus.OK)
 
-        assert response.json["api_key"]
-        assert len(response.json["api_key"]) == 2
-        assert response.json["api_key"]["1"]["name"] == "drb_key"
-        assert response.json["api_key"]["1"]["key"] == pbench_drb_api_key.api_key
-        assert response.json["api_key"]["2"]["name"] == "secondary_key"
-        assert (
-            response.json["api_key"]["2"]["key"] == pbench_drb_secondary_api_key.api_key
-        )
+        assert response.json[0]["id"] == pbench_drb_api_key.id
+        assert response.json[0]["name"] == pbench_drb_api_key.name
+        assert response.json[0]["key"] == pbench_drb_api_key.api_key
```

Comment:
Here for this GET method, after querying we're using `as_json` method and returning a response. But in `as_json` function we aren't returning any information regarding `user` like `user.id` or `user.username` . So I didn't assert any `user`. 

Should I add a `user` field as well in `as_json` method ?

Reply:
From what I see, we use the result of `as_json()` to respond to the `GET` and `POST` requestes as well as for auditing the `POST` (and, hopefully, soon, the `DELETE`) request(s).  In terms of the response, I suppose that the `user` value isn't all that useful, because the client will presumably know what its `user` is; however, in terms of a human examining the audit log, I suspect that we would very much appreciate having direct access to the `user`, rather than having to manually look it up using the API key which might no longer exist.

So, yes, I think you should add the `user` field (or, perhaps `user.username` or something with a human-readable value) to the return from `as_json()`.  That way, we'll have it in the audit log and we can use it in the unit tests.